src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import _ from 'underscore';
 import lodashGet from 'lodash/get';
+import lodashFilter from 'lodash/filter';
+
```

Comment:
You can remove these lines as well since we are no longer using `lodashFilter`.

Reply:
@sobitneupane please check again.

Reply:
Thanks for the quick update.